README.md

```diff
@@ -1 +1,18 @@
 # docker-terraform
+
+This container allows to run Terraform ( 0.6.7 ) inside docker container. You need to export your credencials as envirnoment variables. For AWS it needs ```TF_VAR_AWS_ACCESS_KEY_ID``` and ```TF_VAR_AWS_SECRET_ACCESS_KEY```
+
+## How to run and build
+
+- ```git clone https://github.com/combor/docker-terraform.git```
```

Comment:
This refers to your own account. I think we could probably omit the clone/cd instructions altogether.